lib/QubitFlatfileImport.class.php

```diff
@@ -1522,15 +1523,14 @@ public function createOrUpdateNote($typeId, $text, $options = array())
    */
   private function checkNoteExists($objectId, $typeId, $content, $culture)
   {
-    $c = new Criteria;
-    $c->add(QubitNote::OBJECT_ID, $objectId);
-    $c->add(QubitNote::TYPE_ID, $typeId);
-
-    $c->addJoin(QubitNote::ID, QubitNoteI18n::ID);
-    $c->add(QubitNoteI18n::CONTENT, $content);
-    $c->add(QubitNoteI18n::CULTURE, $culture);
-
-    return null !== QubitNote::getOne($c);
+    $query = "SELECT n.id FROM note n
+      INNER JOIN note_i18n i ON n.id=i.id
+      WHERE n.object_id=?
+      AND n.type_id=?
+      AND i.content=?
+      AND i.culture=?";
+    $statement = self::sqlQuery($query, array($objectId, $typeId, $content, $culture));
+    return false !== $statement->fetch(PDO::FETCH_OBJ);
```

Comment:
I've made an additional change here.  This was comparing to null before which is incorrect - PDO fetch returns 'false' when no records are found.